clients/ui/frontend/src/shared/utilities/utils.ts

```diff
@@ -0,0 +1,17 @@
+export const asEnumMember = <T extends object>(
```

Comment:
I feel like we already have this somewhere else :thinking:

Reply:
I think is a refactor, cause it was in a wrong utils file, but let me check